pages/login/profile/[id].js

```diff
@@ -0,0 +1,126 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { getSession } from "next-auth/react";
+import styled from "styled-components";
+import { StyLi } from "@/pages";
+import BrewMethodsForm from "@/components/BrewmethodsForm";
+import RoastDetailCardProfile from "@/components/RoastDetailCardProfile";
+import BrewMethod from "@/components/Methods";
+import useSWR from "swr";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+export default function DetailProfile() {
+  const [edit, setEdit] = useState(false);
+  const [rateEdit, setRateEdit] = useState(false);
+  const router = useRouter();
+  const { id } = router.query;
+  const { data, isLoading, mutate } = useSWR(`/api/user/${id}`, fetcher);
+
+  if (isLoading) return <h1>Loading</h1>;
+  if (data === undefined) return <h1>Den Kaffee gibts wohl nicht ...</h1>;
+  async function addBrewMethod(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    data.roastIdForMethod = id;
+
+    const res = await fetch(`/api/user/${id}`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(data),
+    });
+
+    if (res.status === 418) {
+      return alert("I am watching you, Ernst!");
```

Comment:
Hahaha :D But will this ever happen that the status code `418` is returned? 🐧

Reply:
Ah damn! I returned the wrong code in the API. Changed it accordingly. Ernst kind of hacked my last input :D